data/json/items/tool/lighting.json

```
@@ -232,6 +232,178 @@
     },
     "flags": [ "LIGHT_300", "CHARGEDIM", "TRADER_AVOID", "BELT_CLIP", "WATER_BREAK", "HELMET_HEAD_ATTACHMENT", "HEAD_STRAP_MOUNT" ]
   },
+  {
+    "id": "compact_tac_flashlight",
+    "type": "TOOL",
+    "//": "Based on the TLR-1, this can go on a pistol or a long-arm",
+    "looks_like": "flashlight",
+    "name": { "str": "compact tactical flashlight (off)", "str_pl": "compact tactical flashlights (off)" },
```

```suggestion
    "name": { "str": "compact weaponlight (off)", "str_pl": "compact weaponlights (off)" },
```
I think this is more succinct.